client/coral-framework/actions/assets.js

```diff
@@ -0,0 +1,19 @@
+import coralApi from '../helpers/response';
+import {addItem} from './items';
+
+export const FETCH_MULTIPLE_ASSETS = 'FETCH_MULTIPLE_ASSETS';
+export const RECEIVE_MULTIPLE_ASSETS = 'RECEIVE_MULTIPLE_ASSETS';
+export const FAILURE_MULTIPLE_ASSSETS = 'FAILURE_MULTIPLE_ASSSETS';
+
+export const fetchMulitpleAssets = ids => {
+  return dispatch => {
+    dispatch({type: FETCH_MULTIPLE_ASSETS});
```

Comment:
We are using `_REQUEST` `_SUCCESS` `_FAILURE` as postfix

Can we change those?